elements/lisk-chain/src/data_access/cache/block.ts

```diff
@@ -18,8 +18,11 @@ import { BlockHeader } from '../../types';
 import { Base } from './base';
 
 export class BlockCache extends Base<BlockHeader> {
-	public constructor(size: number = 500) {
-		super(size);
+	public constructor(
+		minCachedItems: number = 303,
```

Comment:
Since defualt is defined in the `chain.ts` I think this constructor doesn't need the default value

Reply:
Done.